test/editor/serialize-test.ts

```diff
@@ -94,5 +94,11 @@ describe('editor/serialize', function() {
             const html = htmlSerializeIfNeeded(model, { useMarkdown: false });
             expect(html).toBe('\\*hello\\* world &lt; hey world!');
         });
+        it('plaintext remains plaintext even when forcing html', function() {
```

Comment:
missing a blank line above this:
```suggestion

        it('plaintext remains plaintext even when forcing html', function() {
```